### ISSUE TITLE
Warn Users That Sumatra Is Not Accessible To Screen Readers

### ISSUE THREAD

**Issue by fclorenzo** [KenneyNL/Adobe-Alternatives#47]
In search for an Acrobat alternative, I found Sumatra, which is already a very well established program, with years of development. I remember trying it like almost 10 years ago and having to drop it because it wasn't accessible. The other day, i decided to give it another chance to see if anything changed in terms of accessibility, and, to my surprise, the answer was no. It really feels surprising that such a well known, liked and established software still doesn't follow basic software development guidelines at this day, but it is what it is. I just ask you guys to warn users about this fact then.
P.S. If you want more info about this, you can check this:
<https://nvda.groups.io/g/nvda/topic/accessibility_in_sumatrapdf/755127>
